src/smo/data/note.ts

```diff
@@ -363,11 +369,9 @@ export class SmoNote implements Transposable {
     this.flagState = (this.flagState + 1) % 3;
   }
 
+  //todo: double check this
   get dots() {
-    if (this.isTuplet) {
-      return 0;
-    }
-    const vexDuration = SmoMusic.closestSmoDurationFromTicks(this.tickCount);
+    const vexDuration = SmoMusic.closestSmoDurationFromTicks(this.stemTicks);
```

Comment:
I see now that dots in tuplets aren't handled correctly in the main branch.  This kind of gets back to the note not knowing enough about the tuplet it's in to make this determination, and why I originally had the 'stemTicks' calculated outside of SmoNote.  We'll need to think about this.